components/patient-data/ui/src/main/resources/PhenoTips/UIX_Field__date_of_death.xml

```diff
@@ -111,7 +111,22 @@
       <content>{{include document="PhenoTips.PatientSheetMacros" /}}
 
 {{velocity}}
-#__displayFormatted('2-col' 'date_of_death' '' 'date' '')
+#__hasDataToDisplay('date_of_death', $showDod)
+#__hasBooleanDataToDisplay('date_of_death_unknown' $showDodu)
+#if ($showDod || $showDodu)
+  (% class="controlled-group complete-hide" %)(((
+  #if ($showDod)
+    (% class="controlled" %) (((
+    #__displayFormatted('2-col' 'date_of_death' '' 'date' '')
+    )))
+  #end
+  #if ($showDodu)
+    (% class="controller" %)(((
+    #__displayFormatted('2-col' 'date_of_death_unknown')
```

Comment:
Checkboxes and radio buttons should always be displayed _before_ their label. Our form layout is pretty hard to read already with all the text inputs on the far right. Using this structure for checkboxes will practically ensure that the user doesn't see the checkbox at all most of the time :)

Please see other checkboxes in the form (e.g. "This patient is clinically normal") and use a layout consistent with those.This definitely needs to be fixed before we can close the issue.